packages/server/graphql/public/typeDefs/User.graphql

```diff
@@ -275,7 +275,12 @@ type User {
   """
   Get the list of all organizations a user belongs to
   """
-  organizations: [Organization!]!
+  organizations(
+    """
+    Filter by organization id
+    """
+    id: ID
+  ): [Organization!]!
```

Comment:
-1 I think as we only allow for 1 id input, there should be a separate singular field `organization(id: ID!)` like we have `teams` and `team` on `User`

Reply:
@Dschoordsch make sense, just noticed we already had it implemented, fixed!